lib/server.js

```diff
@@ -130,20 +130,20 @@ module.exports = (config) => {
     return registrationMan(server, config)
         .then(() => {
             // Initialize common data in buildFactory and jobFactory
-            if (server.app.buildFactory) {
```

Comment:
how come we no longer need if statement?

Reply:
`buildFactory` and `jobFactory` are always required for Screwdriver to work, so I think these if statements are no longer need. Please teach me if these statements are still need.